package.json

```diff
@@ -33,7 +33,8 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "^5.1.0",
-    "@openzeppelin/contracts-upgradeable": "^5.1.0"
+    "@openzeppelin/contracts-upgradeable": "^5.1.0",
+    "forge": "^2.3.0"
```

Comment:
we shouldn't need a foundry/forge javascript package

Reply:
This project seems to have a foundry dependency already (mentioned in README)  so we shouldn't need this package if you have that installed. Then the following yarn commands will run the tests using that foundry installation.

```json
  "test": "forge test -vvv",
  "test:integration": "forge test --match-contract Integration -vvv",
  "test:symbolic": "halmos",
  "test:unit": "forge test --match-contract Unit -vvv",
  "test:unit:deep": "FOUNDRY_FUZZ_RUNS=5000 yarn test:unit"
```